From 0bfade685783ccd193b2e1b94d0e4ef5ce449432 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Wed, 20 Nov 2013 17:52:50 +0800
Subject: [PATCH] remove the test to convert euc-jp

Remove the test "Test against HP-UX 11.11 bug:
No converter from EUC-JP to UTF-8 is provided"
since we don't support HP-UX and it causes
guile-native compile failure if the euc-jp is not
installed on the host

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 iconv.m4 | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/iconv.m4 b/iconv.m4
index a503646..dbe7ca1 100644
--- a/iconv.m4
+++ b/iconv.m4
@@ -159,17 +159,6 @@ int main ()
       }
   }
 #endif
-  /* Test against HP-UX 11.11 bug: No converter from EUC-JP to UTF-8 is
-     provided.  */
-  if (/* Try standardized names.  */
-      iconv_open ("UTF-8", "EUC-JP") == (iconv_t)(-1)
-      /* Try IRIX, OSF/1 names.  */
-      && iconv_open ("UTF-8", "eucJP") == (iconv_t)(-1)
-      /* Try AIX names.  */
-      && iconv_open ("UTF-8", "IBM-eucJP") == (iconv_t)(-1)
-      /* Try HP-UX names.  */
-      && iconv_open ("utf8", "eucJP") == (iconv_t)(-1))
-    result |= 16;
   return result;
 }]])],
         [am_cv_func_iconv_works=yes],
--
1.8.3

